src/common/src/types/mod.rs

```diff
@@ -136,6 +136,9 @@ pub enum DataType {
     #[display("serial")]
     #[from_str(regex = "(?i)^serial$")]
     Serial,
+    #[display("int256")]
+    #[from_str(regex = "(?i)^int256$")]
```

Comment:
I recall we would like to call this sth like `rw_int256`. Because PostgreSQL already uses `int2`/`int4`/`int8` to mean bytes rather than bits, and `int256` would be inconsistent while `int32` would be confusing.

cc @TennyZhuang @lmatz